Character.cpp

```diff
@@ -27,6 +27,8 @@ void Character::attack( Character& other )
     isDefending = false;
     std::cout << getName() << " has attacked " << other.getName() << std::endl;
     //subtract attackDamage from other->hitPoints
+    other.hitPoints -= attackDamage;
+    
     if( other.takeDamage(attackDamage) <= 0 )
```

Comment:
Don't add anything to this function.  the comment is explaining what `other.takeDamage()` does.